pkg/util/provider/machinecontroller/machine_test.go

```diff
@@ -241,6 +241,13 @@ var _ = Describe("machine", func() {
 		objMeta := &metav1.ObjectMeta{
 			GenerateName: "class",
 			Namespace:    testNamespace,
+			Finalizers:   []string{MCMFinalizerName},
+		}
+
+		objMetaWithoutFinalizer := &metav1.ObjectMeta{
+			GenerateName: "class",
+			Namespace:    testNamespace,
+			Finalizers:   []string{MCMFinalizerName},
```

Comment:
I think `Finalizers` field shouldn't be a part of this

Reply:
Good catch! Thank you